### PR TITLE
added wishlist

### PR DESCRIPTION
**Pull request by swstica**

### CODE REVIEW

**Review by malaykhakhar — APPROVED** [review]
The code looks good. You can merge it.

File: wishlist/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Document</title>
+</head>
+<body>
+
+    <header>
+        <div class="navbar">
+            <div class="nav-items flex-flow-row">
+                <div class="nav-item"><a><img src="../images/logo.png" height="40rem" width="100rem"></a></div>
+                <div class="nav-item"><a href="../listing-page/listingpage.html">Categories</a></div>
```
Comment (omkarpatke) [inline wishlist/wishlist.html:17]: here you want to avoid inline CSS.

File: wishlist/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        <div class="card-container">
+            <h2 class="wishlist-list">My wishlist</h2>
+            <div class="flex-flow-row card-items">
+
+                <div class="flex-flow-row product-cards">
+                    <div class="card card-component">
+                        <img class="card-img" src="../images/women.jpg">
+                        <span class="dismiss"><i class="fa fa-times fa-2x"></i></span>
+                        <div class="card-details flex-flow-column">
+                            <h2><a>Apparel</a></h2>
```
Comment (rahulk31) [inline wishlist/wishlist.html:47]: Suggestion is to wrap the h2 with a tag.

File: wishlist/wishlist.html [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+                        <div class="card-actions">
+                            <button class=" card-action-btn Move to card-btn">Move to cart</button>
+                        </div>
+                    </div>
+    
+                    <div class="card card-component">
+                        <img class="card-img" src="../images/women.jpg">
+                        <span class="dismiss"><i class="fa fa-times fa-2x"></i></span>
+                        <div class="card-details flex-flow-column">
+                            <h2><a>Apparel</a></h2>
```
Comment (rahulk31) [inline wishlist/wishlist.html:60]: Similarly here and other places as well.

File: listing-page/listingpage.css [hunk 4]
```diff
@@ -76,10 +76,11 @@ section{
 /*cards*/
```
Comment (yaswanthmyneni) [inline listing-page/listingpage.css:76]: changes: you have to use CSS variables for color.
Screenshot 2022-02-20 at 2 24 23 PM

File: index.css [hunk 5]
```diff
@@ -64,8 +64,8 @@ a:link {
     margin-left: -5px;
     font-size: 1rem;
     border: none;
-    width: 20rem;
-    padding-left: 5rem;
+    width: 24rem;
+    
     background-color: var(--grey-color);
     border-top-right-radius: 0.5rem;
     border-bottom-right-radius: 0.5rem;
```
Comment (yaswanthmyneni) [inline index.css:71]: changes: use px units for less than 1rem.

**Review by rahulk31 — APPROVED** [review]
Overall code looks good. Liked the way you have made use of your Component Library and used very little explicit css. Can be merged.

**Review by yaswanthmyneni — CHANGES_REQUESTED** [review]
UI looks good, you can merge after changes.

**Review by omkarpatke — CHANGES_REQUESTED** [review]
Your Code Looks Good, do changes that I suggest, and feel free to merge.

**Comment by prankurpandeyy** [comment]
LOoks good give some padding and margin to card so that can be look seperated .

**Review by prankurpandeyy — APPROVED** [review]

File: index.css (hunk 5 above)
Comment (swstica) [reply index.css:71]: I tried but then it exceeds 10px, and using px for >10 is also not good.

**Pull request merged**